python/cells.cpp

```diff
@@ -321,6 +360,36 @@ void register_cells(pybind11::module& m) {
         .def("__repr__", [](const label_dict_proxy& d){return d.to_string();})
         .def("__str__",  [](const label_dict_proxy& d){return d.to_string();});
 
+    // arb::cv_policy wrappers
+
+    pybind11::class_<arb::cv_policy> cv_policy(m, "cv_policy",
+            "Describes the rules used to discretize (compartmentalise) a cable cell morphology.");
+    cv_policy
+        .def_property_readonly("domain",
+                               [](const arb::cv_policy& p) {return util::pprintf("{}", p.domain());},
+                               "The domain on which the policy is applied.")
+        .def(pybind11::self + pybind11::self)
+        .def(pybind11::self | pybind11::self)
```

Comment:
Does this just call the underlying methods in C++?

Reply:
Yes: it is a shorthand way to specify that the operator overloads on the C++ side can be used in Python.